packages/@lwc/shared/src/html-attributes.ts

```diff
@@ -13,13 +13,13 @@ const CAMEL_REGEX = /-([a-z])/g;
  * Maps boolean attribute name to supported tags: 'boolean attr name' => Set of allowed tag names
  * that supports them.
  */
-const BOOLEAN_ATTRIBUTES = new Map([
-    ['autofocus', new Set(['button', 'input', 'keygen', 'select', 'textarea'])],
-    ['autoplay', new Set(['audio', 'video'])],
-    ['checked', new Set(['command', 'input'])],
+const BOOLEAN_ATTRIBUTES = /*@__PURE__@*/ new Map([
```

Comment:
Needed because rollup started marking `Map`, `Set` etc. as impure. https://github.com/rollup/rollup/pull/4955

We could mark Map, Set as pure in rollup.config.js, but that would be on the consumer to do, and if they, for some reason, don't do it, these are not tree-shaken.

Reply:
Interesting! Great find. Looks like they're marking all Maps/Sets as a temporary fix for https://github.com/rollup/rollup/issues/4282 . In any case, we can keep the `/*@__PURE__@*/` permanently to play it safe.